app/views/providers/details.html.erb

```diff
@@ -36,6 +36,9 @@
       <dl class="govuk-summary-list">
         <%= enrichment_summary_item('About your organisation', @provider.train_with_us, 'train_with_us') %>
         <%= enrichment_summary_item('Training with disabilities and other needs', @provider.train_with_disability, 'train_with_disability') %>
+        <% @provider.accrediting_providers.each do |provider| %>
```

Comment:
```suggestion
        <% @provider.accrediting_providers.each do |accrediting_provider| %>
```

`get_provider_name(provider['UcasProviderCode']` still think backend should sort it all out for you